.github/workflows/test.yml

```diff
@@ -26,7 +26,10 @@ jobs:
             - name: Checkout code
               uses: actions/checkout@v2
             - name: Format  
-              run: if [ ! -z "$(find . -type f -name "*.go" -not -path "./vendor/*" | sed "s|^\./||" | xargs gofmt -l)" ]; then exit 1; fi
+              run: if ! [ -z "$(find . -type f -name "*.go" -not -path "./vendor/*" | sed "s|^\./||" | xargs gofmt -l)" ]; then
+                     echo  -e " $(find . -type f -name "*.go" -not -path "./vendor/*" | sed "s|^\./||" | xargs gofmt -l)";
```

Comment:
Please fix the gofmt check.

Reply:
The above one works fine, I verified it from old travis and in 0.11TF workflow I tested it and it was printing out the unformatted files.